cinema/serializers.py

```diff
@@ -1 +1,88 @@
-# write serializers here
+from rest_framework import serializers
+from .models import (
+    CinemaHall,
+    Movie,
+    MovieSession,
+    Actor,
+    Genre
+)
```

Comment:
Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) Read https://peps.python.org/pep-0008/#imports